src/jira/connect.ts

```diff
@@ -78,6 +78,14 @@ export default async (_: Request, res: Response): Promise<void> => {
 					},
 					homeUrl: "https://github.com/features/actions",
 				},
+				jiraRemoteLinkInfoProvider: {
+					key: "github-remotelinks-integration",
+					name: {
+						value: "GitHub"
+					},
+					logoUrl: "https://github.githubassets.com/images/modules/logos_page/GitHub-Mark.png",
+					homeUrl: "https://github.com/features/security"
```

Comment:
I wonder if we would have more remote links from GitHub? (like GitHub Issues?) 
Should we put more generic URL like or APP repo URL or GitHub homepage?

Reply:
Ah yep, that's a good point. I'll switch it to the GitHub homepage so that it's more generic in case of other future uses of remote links feature